pkg/bff/gds.go

```diff
@@ -104,7 +104,16 @@ func (s *Server) Lookup(c *gin.Context) {
 			continue
 		}
 
-		data, err := wire.Rewire(result)
+		var lookup *gds.LookupReply
+		var ok bool
+		if lookup, ok = result.(*gds.LookupReply); !ok {
+			err := fmt.Errorf("unexpected result type: %T", result)
+			log.Error().Err(err).Msg("unexpected result type")
+			c.JSON(http.StatusInternalServerError, api.ErrorResponse(err))
+			return
+		}
+
+		data, err := wire.Rewire(lookup)
```

Comment:
I changed `ParallelGDSRequests` to return an `interface{}` slice, which does reduce some code in `parallel.go` but does require the caller to type assert the results. Should we want to revert the return type back to a `protoMessage` since the GDS and members will always return that anyway?

Reply:
That makes sense, thanks